### PR TITLE
chore: add make release

### PR DESCRIPTION
**Pull request by xrkffgg**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [x] Other (about what?)

### 💡 Background and solution
将 ant-bot 中 relase 发布功能 移动到此

https://github.com/xrkffgg/test-ci/releases  

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |  -         |
| 🇨🇳 Chinese |      - |
### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: .github/workflows/make-release.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+on:
+  create
+
+jobs:
+  make-release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+
```
Comment (xrkffgg) [inline .github/workflows/make-release.yml:11]:
```suggestion

```

File: .github/workflows/make-release.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    steps:
+      - uses: actions/checkout@master
+      - name: install
+        run: yarn
+
+      - name: node js
+        run: node ./script/make-release.js
+        env:
+          GH_TOKEN: ${{ secrets.ANT_BOT_TOKEN }}
+          DD_TOKEN: ${{ secrets.DINGDING_BOT_TOKEN }}
```
Comment (xrkffgg) [inline .github/workflows/make-release.yml:18]: 这个地方 还需要 大佬们 配置下

Comment (yesmeck) [reply .github/workflows/make-release.yml:18]: done

**Review by kerm1it — APPROVED** [review]

**Comment by xrkffgg** [comment]
image

这报错，我也是醉了，是不让用解构的么

**Comment by xrkffgg** [comment]
先等等，我晚上试试能不能新弄个action来

**Comment by xrkffgg** [comment]
antd 的 node_modules 太大，我试试用个简单哒

**Pull request closed without merging**